client.go

```diff
@@ -45,15 +45,15 @@ func NewFromAddr(addr string) (*Client, error) {
 
 // NewFromURL creates a new bloomd client from URL struct
 func NewFromURL(u *url.URL) (*Client, error) {
-	conn, err := createSocket(u)
+	conn, err := Connect(u)
 	if err != nil {
 		return nil, err
 	}
 
 	return NewFromConn(conn)
 }
 
-func createSocket(u *url.URL) (net.Conn, error) {
+func Connect(u *url.URL) (net.Conn, error) {
```

Comment:
Could you add a comment for this exported method? :)

Reply:
didn't we have some golintci bot here enabled before? If it's disabled, then we probably can just add regular golintci as a tool

Reply:
@antnsi It is enabled, it's just hidden down there when all the checks have passed. This repo doesn't have our golangci config, so it didn't complain about the missing comment.